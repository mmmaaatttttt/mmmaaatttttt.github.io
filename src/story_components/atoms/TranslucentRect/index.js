import styled from "styled-components";

const TranslucentRect = styled.rect`
  opacity: 0.7;

  &:hover,
  &:active {
    opacity: 1;
  }
`;

export default TranslucentRect;
