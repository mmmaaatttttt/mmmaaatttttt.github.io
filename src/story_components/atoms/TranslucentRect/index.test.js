import React from "react";
import { render } from "@testing-library/react";
import TranslucentRect from "./";

it("renders without crashing", () => {
  render(<TranslucentRect />);
});

it("matches snapshot with props and children", () => {
  const { asFragment } = render(<TranslucentRect />);
  expect(asFragment()).toMatchSnapshot();
});
