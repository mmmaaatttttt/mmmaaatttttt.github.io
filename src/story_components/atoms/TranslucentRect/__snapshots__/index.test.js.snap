// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot with props and children 1`] = `
<DocumentFragment>
  <rect
    class="sc-bdVaJa bELJdd"
  />
</DocumentFragment>
`;
